wiki/docs/running.md

```diff
@@ -0,0 +1,34 @@
+Running
+---
+
+We provide a Python script, i.e. `launch`, that simplifies running Mimiker OS.
+The kernel image is run with QEMU or Renode simulator. One or more serial
+consoles are available for interaction. Optionally you can attach to simulator
+with `gdb` debugger.  All of that is achieved by running all interactive
+sessions within [tmux](https://github.com/tmux/tmux/wiki) terminal multiplexer
+with default key bindings.
+
+In project main directory, run command below that will start the kernel in
```

Comment:
s/main/root/ imho (you already use root in building.md)

Reply:
I'd rephrase this sentence: "To start kernel in test-run mode, run the following command in the project's root directory:
<command here>
To finish simulation [...]